### DISCUSSION TITLE
pdf tempalte: how to change $(record : createdtime)$ to a [date]

### DISCUSSION THREAD

**Discussion by newminder** [YetiForceCompany/YetiForceCRM#16988] [How to] [answered]
pdf tempalte: how to change $(record : createdtime)$ to a [date], I tried the function date('y-m-d', $(record : createdtime)$), it does not work, if anyone knows, please help me.

**Comment by rskrzypczak** [comment, marked as answer]
You can try to use twig: https://twig.symfony.com/doc/3.x/filters/date.html
`{{ YFParser("$(record : createdtime)$")|date("m/d/Y") }}`
but whether this will work correctly depends on the format you have set in your user configuration

**Reply by newminder** [reply]
thank you very much, this works exactly as I need.  appreciate.
